studio: display tags on all cards of the main feed

## Description

Currently on the main feed ("the home screen") of the editor ("studio"), the tags (hashtags like `#javascript #tech` etc.) are only displayed on the first card of the list.

I think it would be cool to display them on each card.

### Current display

First card:

Screenshot
Title
Author + date
Description
Tags

Other cards

Screenshot
Title
Author + date

### Goal

First card:

Screenshot
Title
Author + date
Description
Tags

Other cards

Screenshot
Title
Author + date
Tags <----------- TODO

Hi @peterpeterparker,  I would like to work on this issue. I got a hiccup while building DeckDeckGo Studio.
while running  `npm run start` I am getting the following error:

```
[ ERROR ]  Cannot find module './config.dev.json' Require stack: -
           /home/github/deckdeckgo/studio/stencil.config.ts -
           /home/github/deckdeckgo/studio/node_modules/@stencil/core/dist/sys/node/index.js -
           /home/github/deckdeckgo/studio/node_modules/@stencil/core/bin/stencil Error: Cannot find module
           './config.dev.json' Require stack: - /home/github/deckdeckgo/studio/stencil.config.ts -
           /home/github/deckdeckgo/studio/node_modules/@stencil/core/dist/sys/node/index.js -
           /home/github/deckdeckgo/studio/node_modules/@stencil/core/bin/stencil at
           Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15) at Function.Module._load
           (internal/modules/cjs/loader.js:677:27) at Module.require (internal/modules/cjs/loader.js:830:19) at require
           (internal/modules/cjs/helpers.js:68:18) at Object.<anonymous>
           (/home/github/deckdeckgo/studio/stencil.config.ts:14:41) at Module._compile
           (internal/modules/cjs/loader.js:936:30) at Object.require.extensions..ts
           (/home/github/deckdeckgo/studio/node_modules/@stencil/core/dist/sys/node/index.js:4817:16) at
           Module.load (internal/modules/cjs/loader.js:790:32) at Function.Module._load
           (internal/modules/cjs/loader.js:703:12) at Module.require (internal/modules/cjs/loader.js:830:19)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

--- a/studio/src/app/components/feed/card/app-feed-card/app-feed-card.scss
+++ b/studio/src/app/components/feed/card/app-feed-card/app-feed-card.scss
@@ -16,6 +16,7 @@ app-feed-card {
       }
 
       ion-card-title {
+        display: none;
         white-space: nowrap;
         overflow: hidden;
         font-size: var(--font-size-h1);

--- a/studio/src/app/components/feed/card/app-feed-card/app-feed-card.scss
+++ b/studio/src/app/components/feed/card/app-feed-card/app-feed-card.scss
@@ -6,7 +6,7 @@ app-feed-card {
     ion-card-content {
 
       &.compact {
-        app-feed-card-tags, p.content {
+        p.content {
           display: none;
         }
       }

